tests/collecting/CollectorsTest.py

```diff
@@ -387,38 +387,12 @@ def test_get_all_bears(self):
                 assert issubclass(bear, Bear)
             self.assertSetEqual(
                 {b.name for b in bears},
-                {'DependentBear',
-                 'EchoBear',
-                 'LineCountTestBear',
-                 'JavaTestBear',
-                 'SpaceConsistencyTestBear',
-                 'TestBear',
-                 'ErrorTestBear',
-                 'RaiseTestBear',
-                 'TestDepBearA',
-                 'TestDepBearBDependsA',
-                 'TestDepBearCDependsB',
-                 'TestDepBearAA',
-                 'AspectTestBear',
-                 'TestDepBearDependsAAndAA'})
+                set(TEST_BEAR_NAMES))
```

Comment:
Why `set()` is used?

Reply:
The other half of assertEqual is a set.

But, I should change that now.

Reply:
But get_all_bears is unsorted..?

Reply:
I could use assertCountEqual , but that doesnt strengthen the assertion at all.

Reply:
Yes `get_all_bears` is unsorted.

Reply:
So set comparion is needed..?

If I am missing something ... yell out.
Otherwise .. need to merge

Reply:
Yeh. Right.